frontend/src/lib/forms/superforms.ts

```diff
@@ -5,7 +5,8 @@ import type { SuperValidated, ZodValidation } from 'sveltekit-superforms';
 import { superValidateSync } from 'sveltekit-superforms/client';
 import type { AnyZodObject, z } from 'zod';
 import type { ErrorMessage } from './types';
-import { randomFormId } from '.';
+import { randomFormId } from './utils';
+// NOTE: Don't import randomFormId from '.' as that creates a circular module dependency in any component that imports lexSuperForm
```

Comment:
I don't think the comment is a valuable addition. The reason being that we probably have 100 imports that are just as vulnerable.
Ultimately, this is just something we have to be aware of 🤷.

Reply:
Removed in commit f002ab6.